kubernetes/samples/scripts/create-soa-domain/common/createSOADomain.py

```diff
@@ -0,0 +1,605 @@
+# Copyright 2018, 2019, Oracle Corporation and/or its affiliates. All rights reserved.
+# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
+
+import os
+import sys
+
+import com.oracle.cie.domain.script.jython.WLSTException as WLSTException
+
+class SOA12213Provisioner:
+
+    jrfDone = 0;
+    MACHINES = {
+        'machine1' : {
+            'NMType': 'SSL',
+            'ListenAddress': 'localhost',
+            'ListenPort': 5658
+        }
+    }
```

Comment:
I assume this isn't interfering with the NM configuration that the Operator overrides?